### PR TITLE
fix(curriculum): Bad grammar cat photo app 8

### LINKED ISSUE #49964: Bad Grammar Cat Photo App 8
### Describe the Issue

The last sentence here is bad grammar. Also. The period after the code block can look awkward on mobile.

> HTML attributes are special words used inside the opening tag of an element to control the element's behavior. The src attribute in an img element specifies the image's URL (where the image is located). An example of an img element using an src attribute: ``.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-8

### Your code

N/A

### Expected behavior

Rework sentence for better grammar and formatting.

Possible fix

> HTML attributes are special words used inside the opening tag of an element to control the element's behavior. For example, the src attribute in an img element specifies the image's URL (where the image is located). 

```
<img src="https://www.example.com/the-image.jpg">
```

### Screenshots

_No response_

### System

All

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by cto234**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #49964

Before:
Screen Shot 2023-04-05 at 1 30 45 PM
After:
Screen Shot 2023-04-05 at 1 31 03 PM

This fix cleans up formatting and grammar to make lesson more readable.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md [hunk 1]
```diff
@@ -7,11 +7,13 @@ dashedName: step-8
...
-HTML <dfn>attributes</dfn> are special words used inside the opening tag of an element to control the element's behavior. The `src` attribute in an `img` element specifies the image's URL (where the image is located). An example of an `img` element using an `src` attribute: `<img src="https://www.example.com/the-image.jpg">`.
+HTML <dfn>attributes</dfn> are special words used inside the opening tag of an element to control the element's behavior. The `src` attribute in an `img` element specifies the image's URL (where the image is located).
 
-Inside the existing `img` element, add an `src` attribute with this URL:
+Here is an example of an `img` element with a `src` attribute pointing to the freeCodeCamp logo:
 
-`https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg`
+```html
+<img src="https://cdn.freecodecamp.org/platform/universal/fcc_secondary.svg">
+```
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md:16]: The instructions for the step should not be removed.

The example should instead come before the `Inside the existing...` instructions.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md (hunk 1 above)
Comment (cto234) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md]: Whoops! Sorry about that, added those back.

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Review by naomi-lgbt — APPROVED** [review]

**Comment by gikf** [comment]
Hi @cto234. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**